src/components/EditCharacter.js

```diff
@@ -43,6 +44,18 @@ export default class EditCharacter extends Component {
       activeCharacterSubrace = ''
     }
 
+    if (this.props.activeCharacter.hasOwnProperty('klass')) {
+      activeCharacterClass = this.props.activeCharacter.klass
+    } else {
+      activeCharacterClass = ''
+    }
```

Comment:
In JavaScript, the convention here is to simply do 
```
activeCharacterClass = this.props.activeCharacter.klass || ''
```
instead of doing this kind of thing.